src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractPackage`2.cs

```diff
@@ -152,5 +152,15 @@ protected override void Dispose(bool disposing)
         }
 
         protected abstract string RoslynLanguageName { get; }
+
+        private void EnableRemoteHostClientService()
+        {
+            ((RemoteHostClientServiceFactory.RemoteHostClientService)this.Workspace.Services.GetService<IRemoteHostClientService>()).Enable();
```

Comment:
Enable and Disable is host API and GetRemoteHostClientAsync is consumer API.

I could have created yet another interface IRemoteHostClientHostService or something that contains Enable and Disable. but felt that makes unnecessary abstraction. and I wanted that Enable/Disable functionality is not something other people can override in VS Workspace.